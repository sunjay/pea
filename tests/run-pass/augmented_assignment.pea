fn main() {
    let x = 33;
    println!(x);
    x += 1;
    println!(x);
    x -= 21;
    println!(x);
    x *= 9;
    println!(x);
    x /= 2;
    println!(x);
    x %= 5;
    println!(x);
}
