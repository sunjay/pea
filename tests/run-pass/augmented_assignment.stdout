33
34
13
117
58
3
